clinica/utils/input_files.py

```diff
@@ -5,197 +5,445 @@
+        (
+            f"{'right' if hemisphere == HemiSphere.RIGHT else 'left'} white matter/gray matter border "
+            f"surface ({hemisphere.value}.white) generated with t1-freesurfer-longitudinal."
+        ),
+        "t1-freesurfer and t1-freesurfer longitudinal",
+    )
+
+
+def _get_annot_file_name(hemisphere: HemiSphere, parcellation: Parcellation) -> str:
+    if parcellation == Parcellation.DESIKAN:
+        return f"{hemisphere.value}.aparc.annot"
+    if parcellation == Parcellation.DESTRIEUX:
+        return f"{hemisphere.value}.aparc.a2009s.annot"
+
+
+def get_t1_freesurfer_segmentation(parcellation: Parcellation) -> QueryPattern:
+    parcellation = Parcellation(parcellation)
+    filename = (
+        f"aparc{'.a2009s' if parcellation == Parcellation.DESTRIEUX else ''}+aseg.mgz"
+    )
+    return QueryPattern(
+        f"t1/freesurfer_cross_sectional/sub-*_ses-*/mri/{filename}",
+        f"{parcellation.value}-based segmentation (mri/{filename}).",
+        "t1-freesurfer",
+    )
+
+
+def get_t1_freesurfer_parcellation(
+    hemisphere: Union[str, HemiSphere],
+    parcellation: Union[str, Parcellation],
+) -> QueryPattern:
+    hemisphere = HemiSphere(hemisphere)
+    parcellation = Parcellation(parcellation)
+    return QueryPattern(
+        f"t1/freesurfer_cross_sectional/sub-*_ses-*/label/{_get_annot_file_name(hemisphere, parcellation)}",
+        (
+            f"{'left' if hemisphere == HemiSphere.LEFT else 'right'} hemisphere surface-based "
+            f"{parcellation.value} parcellation (label/{_get_annot_file_name(hemisphere, parcellation)})."
+        ),
+        "t1-freesurfer",
+    )
+
+
+def get_t1_freesurfer_template(parcellation: Union[str, Parcellation]) -> QueryPattern:
+    parcellation = Parcellation(parcellation)
+    filename = (
+        f"aparc{'.a2009s' if parcellation == Parcellation.DESTRIEUX else ''}+aseg.mgz"
+    )
+    return QueryPattern(
+        f"freesurfer_unbiased_template/sub-*_long-*/mri/{filename}",
+        f"{parcellation.value}-based segmentation (mri/{filename}) from unbiased template.",
+        "t1-freesurfer-longitudinal or t1-freesurfer-template",
+    )
+
+
+def get_t1_freesurfer_longitudinal_parcellation(
+    hemisphere: Union[str, HemiSphere],
+    parcellation: Union[str, Parcellation],
+) -> QueryPattern:
+    hemisphere = HemiSphere(hemisphere)
+    parcellation = Parcellation(parcellation)
+    return QueryPattern(
+        f"t1/long-*/freesurfer_longitudinal/sub-*_ses-*.long.sub-*_*/label/{_get_annot_file_name(hemisphere, parcellation)}",
+        (
+            f"{'left' if hemisphere == HemiSphere.LEFT else 'right'} hemisphere surface-based "
+            f"{parcellation.value} parcellation (label/{_get_annot_file_name(hemisphere, parcellation)}) "
+            "generated with t1-freesurfer-longitudinal."
+        ),
+        "t1-freesurfer and t1-freesurfer-longitudinal",
+    )
 
 
 def aggregator(func):
```

Comment:
The description for this function needs to be changed a bit since `t1_volume_native_tpm` was removed. I cannot ping you at the right lines with the diff but all the examples here use `t1_volume_native_tpm`.